config/crd/bases/source.toolkit.fluxcd.io_buckets.yaml

```diff
@@ -376,10 +376,6 @@ spec:
               artifact:
                 description: Artifact represents the last successful Bucket reconciliation.
                 properties:
-                  checksum:
```

Comment:
How did this change end up in this PR?

Reply:
Deprecation is part of the move of the `Artifact` from `v1beta2` to `v1`, which means all resources which refer to it are updated as well.